packages/core/src/telemetry/clearcut-logger/clearcut-logger.ts

```diff
@@ -754,6 +763,78 @@ export class ClearcutLogger {
     this.flushIfNeeded();
   }
 
+  logResearchOptInEvent(event: ResearchOptInEvent): void {
+    const data: EventValue[] = [];
+
+    // Add opt_in_status if available
+    if (event.opt_in_status !== undefined) {
+      data.push({
+        gemini_cli_key: EventMetadataKey.GEMINI_CLI_RESEARCH_OPT_IN_STATUS,
+        value: JSON.stringify(event.opt_in_status),
```

Comment:
Can we avoid `JSON.stringify` for these values if the type is a string?

Prefer to just backtick quote here to "cast" to a string (e.g. `${event.opt_in_status}`'

> Why?: JSON.stringify has resulted in a bunch of our logging data being logged as (literal) `"true"` and (literal) `"1234"` with quotes.  This is annoying and makes our backend analysis a bit more complex.

(Yes, I know there are a bunch of other cases where we JSON.stringify things, but it's not good and something we want to fix up.).